js/login.js

```diff
@@ -0,0 +1,89 @@
+import authFormUtils from "./utils/authFormUtils.js";
+
+// DOM elements
+const inputEmail = document.querySelector("#email");
+const inputPw = document.querySelector("#password");
+const detailEmail = document.querySelector(".email-detail");
+const detailPw = document.querySelector(".password-detail");
+const togglePw = document.querySelector(".toggle-visibility");
+const loginBtn = document.querySelector(".login-btn");
+
+// variables
+const inputTags = document.querySelectorAll(".login-form input");
+const verifiationList = (() => {
+  const map = {};
+  inputTags.forEach((el) => {
+    map[el.id] = false;
+  });
+
+  return map;
+})();
+
+// function
+const verifyInputs = () => {
+  authFormUtils.verify(verifiationList)
```

Comment:
```suggestion
  const isValid = authFormUtils.verify(verifiationList)
```

사소하지만, 이런식으로 변수로 만들어주면 함수 내부를 들여다보지않아도 실행 결과값에 대한 예측이 좀 더 쉬워지겠죠? :)